packages/admin-ui/src/i18n/week-data.ts

```diff
@@ -0,0 +1,95 @@
+// Data from https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/weekData.json
+export const weekData: Record<string, number> = {
+  '001': 1,
```

Comment:
What do the numbers mean? Min days? Day of the week that's the first?

Reply:
Day of the week that's the first by locale